cypress/e2e/po/pages/explorer/projects-namespaces.po.ts

```diff
@@ -17,6 +36,10 @@ export default class ProjectNamespacePagePo extends PagePo {
     super(ProjectNamespacePagePo.createPath(clusterId));
   }
 
+  namespaceDetail(clusterId: string, id: string) {
```

Comment:
This isn't so much to do with the project/namespaces list page but a convenience method to construct another page's page PO. Feels like `new ProjectNamespaceDetailPagePo(clusterId, id` could be used on it's own